packages/catalog-realm/catalog-app/components/grid.gts

```diff
@@ -43,6 +43,11 @@ export class CardsGrid extends GlimmerComponent<CardsGridSignature> {
     this.hydratedCardId = cardId;
   }
 
+  //default to rendering 10 skeletons
+  renderSkeletons(length: number = 10) {
+    return Array.from({ length }, (_, i) => i);
+  }
```

Comment:
no usages of renderSkeletons as a function 

make into getter. hardcode the number.

Reply:
👍